src/styles/index.scss

```
@@ -1,3 +1,133 @@
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --text-color-black: #060B35;
+  --text-color-blue: #00ACDC;
+  --text-color-grey: #616070;
+  --text-color-white: #fff;
+}
+
 body {
   margin: 0;
 }
+
+.product {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  margin: 0;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+    &__image {
+      width: 160px;
+      height: 134px;
+      margin: 0 3px 40px;
+    }
+
+    &__name {
+      margin: 0 0 4px;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      color: var(--text-color-black);
+    }
+
+    &__code {
+      margin: 0 0 16px;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: var(--text-color-grey);
+    }
+
+    &__rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    &__price {
+      height: 18px;
+      margin: 24px 0 16px;
```

try to use either margin-bottom or margin-top. Move `margin-bottom: 24px` to `&__rating`
```suggestion
      margin-bottom: 16px;
```